.github/workflows/stale.yml

```
@@ -0,0 +1,15 @@
+---
+name: stale
+
+on:
+  workflow_call:
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          stale-pr-message: 'This PR is stale because it has been open 90 days with no activity. Remove stale label or comment or this will be closed in 7 days.'
+          days-before-stale: 90
+          days-before-close: 7
```

Reading https://github.com/actions/stale?tab=readme-ov-file#list-of-input-options all arguments are optional, but they do have defaults that I think we should override to avoid closing issues
```suggestion
          stale-pr-message: 'This PR is stale because it has been open for 90 days with no activity. Remove stale label or comment or this will be closed in 7 days.'
          days-before-stale: -1 # Do not close issues
          days-before-pr-stale: 90
```

---

All are optional, but:
- stale-issue-message says it's default is unset
- and then there's this:
>           stale-issue-message: 'Message to comment on stale issues. If none provided, will not mark issues stale'
- thus with the past configuration no issues should be touched.

Anyway. Yours might be a safer configuration.

---

Ah thanks for clarifying! Indeed perhaps safest / clearest to be explicit in not closing stale issues